packages/test-app/tests/unit/utils/calendar-event-tooltip-test.js

```diff
@@ -2,13 +2,14 @@ import calendarEventTooltip from 'test-app/utils/calendar-event-tooltip';
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 import { DateTime } from 'luxon';
+import { setupIntl } from 'ember-intl/test-support';
 
 module('Unit | Utility | calendar-event-tooltip', function (hooks) {
   setupTest(hooks);
+  setupIntl(hooks, 'en-us');
```

Comment:
Isn't this unneeded? I thought the calls in https://github.com/ilios/frontend/blob/d392857d2cd5e16d075f85a95ba9b47a520ad101/packages/test-app/tests/helpers/index.js were enough?

Reply:
we're only setting up intl for acceptance and integration tests, not for unit tests. since we only need it there in rare circumstances, like this one.

Reply:
OK, yes. I see that we only run it for two of the three `setup` functions. Thanks!